docs/book/src/cronjob-tutorial/running.md

```diff
@@ -72,6 +72,8 @@ make docker-build docker-push IMG=<some-registry>/<project-name>:tag
 make deploy IMG=<some-registry>/<project-name>:tag
 ```
 
+If you are using an Apple Sillicon M1 Mac, you need to follow the instructions [here](/reference/envtest.md#installation).
```

Comment:
the project does not support arm64 yet.
It is not the only place that would need to receive some tweaks to work with probably.

Reply:
I think it's a very good idea to post this description in the FAQ section!
Can we maintain the documentation that way?

Reply:
We have the wish to create the FAQ for too long and we have other things to add. 
I think that it is also a nice one to go there.